dandi/models.py

```diff
@@ -775,7 +789,7 @@ class CommonModel(DandiBaseModel):
 
     # Linking to this dandiset or the larger thing
     access: List[AccessRequirements] = Field(
-        title="Access Type",
+        title="Access Information",
```

Comment:
```suggestion
        title="Access information",
```
Alternative to above comment on casing "Ethics Approvals" - imho we should not upper case